src/main/scala/temple/builder/ServerBuilder.scala

```diff
@@ -27,34 +27,34 @@ object ServerBuilder {
     detail: LanguageDetail,
     projectUsesAuth: Boolean,
   ): ServiceRoot = {
-    val language = serviceBlock.lookupMetadata[ServiceLanguage].getOrElse(ProjectConfig.defaultLanguage)
-    val database = serviceBlock.lookupMetadata[Database].getOrElse(ProjectConfig.defaultDatabase)
+    val language = serviceBlock.lookupMetadata[ServiceLanguage] getOrElse ProjectConfig.defaultLanguage
+    val database = serviceBlock.lookupMetadata[Database] getOrElse ProjectConfig.defaultDatabase
```

Comment:
why?

Reply:
I was moving the two lines from below (`readable`, `writable`) up, and one of them was infix already, so I made them consistent. Happy to switch them all to methods if you’d prefer

Reply:
I think we've avoided this notation elsewhere

Reply:
fixed